hknweb/events/admin.py

```diff
@@ -41,5 +44,3 @@ def cute_animal(self, request, queryset):
     cute_animal.short_description = "I wanna see a cute animal"
 
 admin.site.register(EventType)
```

Comment:
What if you add a color picker widget in the admin? I guess it's not really necessary though

Reply:
I haven’t found any widget to do so yet, but I have tinkered with the color picker in HTML5, which we do by setting an input attr to color in the form.

I’ll keep looking for this.